media-libs/libggiwmh/libggiwmh-0.3.2-r1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Windows manager hints extensions for libggi"
+HOMEPAGE="https://ibiblio.org/ggicore/packages/libggiwmh.html"
+SRC_URI="mirror://sourceforge/ggi/${P}.src.tar.bz2"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-fbsd ~x86-fbsd"
+IUSE="X"
+
+RDEPEND=">=media-libs/libggi-2.2.2
+	X? ( x11-libs/libXxf86vm
+		x11-libs/libXxf86dga
+		x11-libs/libXext
```

Comment:
keep them sorted

Reply:
don't know why this is still not outdated.. I've updated sorting ;)